app/views/admin/pages/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<% @page_title = 'Edit page' %>
+
+<%= form_for( @page, url: "/admin/page/#{@page.id}", method: 'post' ) do |f| %>
+  <p>
+    Name:
+    <br><%= f.text_field :name %>
+  </p>
+  <p>
+    Description:
+    <br><%= f.text_field :description %>
+  </p>
+  <p>
+    Title:
+    <br><%= f.text_field :title %>
+  </p>
+  <p>
+    Slug:
+    <br><%= f.text_field :slug %>
+  </p>
+  <p>
+    Template:
+    <br><%= select( 'page', 'template_id',
+      PageTemplate.all.collect { |t| [ t.name, t.id ] } ) %>
```

Comment:
Consider using `pluck` for efficiency:
- only queries the db for the used columns
- returns an array of arrays
- avoids cost of constructing `PageTemplate` objects


```diff
-PageTemplate.all.collect { |t| [ t.name, t.id ] }
+PageTemplate.pluck(:name, :id)
```